config/external_name.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2022 Upbound Inc.
+*/
+
+package config
+
+import "github.com/upbound/upjet/pkg/config"
+
+// ExternalNameConfigs contains all external name configurations for this
+// provider.
+var ExternalNameConfigs = map[string]config.ExternalName{
+	// Import requires using a randomly generated ID from provider: nl-2e21sda
+	"null_resource": config.IdentifierFromProvider,
```

Comment:
do you need `null_resource`?

Reply:
I think I can just remove the whole file as the old overrides.go provides the externalName configuration for all resources anyway.